diff --git a/frappe/desk/page/desktop/desktop.js b/frappe/desk/page/desktop/desktop.js
index b53e7e6..c5fea48 100644
--- a/frappe/desk/page/desktop/desktop.js
+++ b/frappe/desk/page/desktop/desktop.js
@@ -559,13 +559,6 @@ class DesktopPage {
 				},
 				order: 30,
 			},
-			{
-				icon: "support",
-				label: "Frappe Support",
-				onClick: function () {
-					window.open("https://support.frappe.io/help", "_blank");
-				},
-			},
 			{
 				icon: "rotate-ccw",
 				label: "Reset Desktop Layout",
