diff --git a/frappe/desk/page/desktop/desktop.js b/frappe/desk/page/desktop/desktop.js
index ba097684660a2797a24a8415f728cff54be0e6a9..2fa95fe97aa2eebc1e909cd708c119191ee74510 100644
--- a/frappe/desk/page/desktop/desktop.js
+++ b/frappe/desk/page/desktop/desktop.js
@@ -455,13 +455,6 @@ class DesktopPage {
 					return frappe.ui.toolbar.show_about();
 				},
 			},
-			{
-				icon: "support",
-				label: "Frappe Support",
-				onClick: function () {
-					window.open("https://support.frappe.io/help", "_blank");
-				},
-			},
 			{
 				icon: "rotate-ccw",
 				label: "Reset Desktop Layout",
