diff --git a/frappe/desk/page/desktop/desktop.js b/frappe/desk/page/desktop/desktop.js
index c5fea48..fcf8953 100644
--- a/frappe/desk/page/desktop/desktop.js
+++ b/frappe/desk/page/desktop/desktop.js
@@ -551,14 +551,6 @@ class DesktopPage {
 				},
 				order: 20,
 			},
-			{
-				icon: "info",
-				label: "About",
-				onClick: function () {
-					return frappe.ui.toolbar.show_about();
-				},
-				order: 30,
-			},
 			{
 				icon: "rotate-ccw",
 				label: "Reset Desktop Layout",
